src/helpers/clipboard/clipboard_helpers.ts

```diff
@@ -54,3 +61,20 @@ export function getPasteZones<T>(target: Zone[], content: T[][]): Zone[] {
     height = content.length;
   return target.map((t) => splitZoneForPaste(t, width, height)).flat();
 }
+
+export function gePastablePluginClipBoardContent(
```

Comment:
pls help. As a wise Lucas once said: "I love pasta but not in my code".

So yeah I wanted to make a difference between the osclipboard content  {key:string} and the useful stuff, that is the html already parsed. It seemed better to only have to invoke a domparser once and  pass around a typed object instead of a string.